File: spec/python/test_str_pad_term_empty.py
import unittest

from str_pad_term_empty import StrPadTermEmpty

class TestStrPadTermEmpty(unittest.TestCase):
    def test_str_pad_term_empty(self):
        r = StrPadTermEmpty.from_file("src/str_pad_term_empty.bin")

        self.assertEqual(r.str_pad, "")
        self.assertEqual(r.str_term, "")
        self.assertEqual(r.str_term_and_pad, "")
        self.assertEqual(r.str_term_include, "@")

File: spec/javascript/test_str_pad_term_empty.js
var assert = require('assert');
var testHelper = require('testHelper');

testHelper('StrPadTermEmpty', 'src/str_pad_term_empty.bin', function(r) {
  assert.equal(r.strPad, '');
  assert.equal(r.strTerm, '');
  assert.equal(r.strTermAndPad, '');
  assert.equal(r.strTermInclude, '@');
});
